horace_core/sqw/@Experiment/Experiment.m

```diff
@@ -93,8 +87,8 @@
             %
             % Each argument can be a single object or array of objects.
             if nargin == 0
-            	% add one null instrument if using the vanilla constructor
-            	% to satisfy the requirements of subsequent initialisation
+                % add one null instrument if using the vanilla constructor
+                % to satisfy the requirements of subsequent initialisation
```

Comment:
This explicitly doesn't do what it says in the comments

Reply:
I've modified Chris's design but left comments intact (people rarely read them). Fixed.